runtime/autoload/provider/python3.vim

```diff
@@ -0,0 +1,31 @@
+" The Python3 provider uses a Python3 host to emulate an environment for running
+" python3-vim plugins. See ":h nvim-provider" for more information.
```

Comment:
All other instances of this use `:help` instead or `:h`, so it should be probably be done here.